packages/frontend/core/src/modules/right-sidebar/entities/right-sidebar.ts

```diff
@@ -1,9 +1,13 @@
 import { LiveData } from '@toeverything/infra/livedata';
 
+import { LocalStorageGlobalState } from '../../infra-web/storage';
 import type { RightSidebarView } from './right-sidebar-view';
 
 export class RightSidebar {
-  readonly isOpen = new LiveData(false);
+  private readonly globalState = new LocalStorageGlobalState();
+  readonly isOpen = new LiveData(
+    Boolean(this.globalState.get<boolean>('app:settings:rightsidebar'))
+  );
```

Comment:
```suggestion
  readonly isOpen = this.globalState.watch<boolean>('app:settings:rightsidebar').map(f => !!(f ?? false));
```

Reply:
`this.globalState.watch(...).map is not a function`